README.md

```diff
@@ -1,3 +1,59 @@
+The name was a coincidence; we just like the ring to it (so did The Behemoth for the real Castle Crashers!)
+Note: the class focused on Game Mechanics, most of the visual and audio assets were used from free libraries or purchased from the Unity Store.
+
+## Overview
+The goal of the game is to traverse deeper and deeper to new levels of the castle, wield a variety of weapons to battle a variety of enemies, utilize gathered loot and coins to get stronger via opening chests, recieving permanent power upgrades, and buying stat & skill upgrades. Once the player reach the end of their journey, usually ended from a gruesome death, they would recieve a score based on their performance in the game.
+Link to our game trailer:
+https://youtu.be/27xrTJ3nenE
+
+### Stats
+Stats could be upgraded between levels and affected the player power in different ways:
+- Strength: increases damage of weapons. 
+- Agility: increases the attack and movement speed of the player.
+- Stamnia: increases player health and the effectiveness of health potions.
+
+### Weapons
+Several weapons were available to the player, most weapons types focusing on different ranged weapons:
+- Sword: a basic weapon that would sweep and do damage in a small range from the player.
+- Rock: a thrown weapon that is easy to target and hit with that does moderate damage.
+- Arrow: a thrown weapon that hits enemies for a moderate amount of damage and leaves a small bleed DoT (damage of time) effect on enemies.
+- Fireball: a small, but hard to hit with, fireball that does a small amount of initial damage but leaves enemies on fire; the on fire DoT ticks for heavy fire damage.
+
+### Skills
+To further modify the player's power, there are four skills to choose from and level up:
+- Stun: modifies melee attacks to stun the enemy for a duration equal to the levels invested in this skill.
+- Peircing: causes each ranged attack to pierce through a number of enemies equal to the levels invested in this skill.
+- Dash: Allows the player access to the Dash ability, with each subsequent invested level past the first reducing the cooldown of the dash ability.
+- Multiply: modifies ranged attacks to fire multiple times based on the level invested into this skill.
+
+## Development
+This project was used as the final project and primary evaluation for the introductory Game Mechanics class (CMPT 306) at the University of Saskatchewan; we had specific goals to achieve to gain a base understanding of game dev. We worked together and delegated various tasks to ensure each team member had work they wanted to complete or had the skills to complete. We often had discussions in weekly meetings, frequent communication via a discord server with a webhook to keep track of work completed and Pull Requests (PR). As a team we were able to support each other through tricky problems and review each others work through feedback and PRs.  
```

Comment:
"frequent communication via a discord server with a webhook to keep track"
to be
"frequent communication via a Discord server with a GitHub webhook to keep track"